charts/presto/templates/configmap-resource-manager.yaml

```diff
@@ -33,7 +33,7 @@ data:
     resource-manager=true
     thrift.server.port=8081
     thrift.server.ssl.enabled=false
-    {{- .Values.config | nindent 4 }}
+    {{- .Values.resourceManager.config | default .Values.config | nindent 4 }}
```

Comment:
Looks like `presto.version` should be added to `configmap-resource-manager.yaml`, otherwise `ha-cluster` mode doesn't work properly. Could you please check it?

Reply:
Yes, confirmed that presto.version is needed for the discovery server to properly detect the Resource Manager. I've Added this.

Also, I noticed we are calling the new version as Presto C++. should we keep the section in the charts as "velox" or should we rename it as "prestocpp" or similar?

Reply:
@tdcmeehan Do you have any strong opinion on how we should name it (`velox`, `prestocpp`, `native` etc)? Otherwise let's keep it `velox` for now.

Reply:
`prestocpp` is the naming we're using and what we should adopt here.